### PR TITLE
Add new mayors

### PR DESCRIPTION
**Pull request by brianhlin**
Per request of @chris-erickson

### CODE REVIEW

File: data/va/municipalities/MaryKatherine-Greenlaw-9fd042ae-f16c-4f87-a3c5-3dd0d229a6ea.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+id: ocd-person/9fd042ae-f16c-4f87-a3c5-3dd0d229a6ea
+name: MaryKatherine Greenlaw
```
Comment (jamesturk) [inline data/va/municipalities/MaryKatherine-Greenlaw-9fd042ae-f16c-4f87-a3c5-3dd0d229a6ea.yml:2]: small thing, but missing a space

**Review by jamesturk — CHANGES_REQUESTED** [review]

File: data/va/municipalities/MaryKatherine-Greenlaw-9fd042ae-f16c-4f87-a3c5-3dd0d229a6ea.yml (hunk 1 above)
Comment (brianhlin) [reply data/va/municipalities/MaryKatherine-Greenlaw-9fd042ae-f16c-4f87-a3c5-3dd0d229a6ea.yml:2]: Good catch @jamesturk. Do you also want the filename updated to `Mary-Katherine-Greenlaw...` as well?

Comment (chris-erickson) [reply data/va/municipalities/MaryKatherine-Greenlaw-9fd042ae-f16c-4f87-a3c5-3dd0d229a6ea.yml:2]: @brianhlin let's just do it 😄

Comment (brianhlin) [reply data/va/municipalities/MaryKatherine-Greenlaw-9fd042ae-f16c-4f87-a3c5-3dd0d229a6ea.yml]: Done. @jamesturk let me know if you want the filename updated, too

**Comment by jamesturk** [comment]
looks like there are a few merge conflicts from the other mayors work, I'll be busy with legislature stuff for the next few days, but I believe this is ready if someone else can fix those

**Pull request merged**